<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QKXSlipTestClass</class>
 <widget class="QMainWindow" name="QKXSlipTestClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QKXSlipTest</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>二维码</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="pushButton_CreateQR">
           <property name="text">
            <string>生成二维码</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>打印控制</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="pushButton_PrePrint">
           <property name="text">
            <string>打印预览</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_Print">
           <property name="text">
            <string>打印</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_Test">
           <property name="text">
            <string>测试</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>封签信息</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="5" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>封签：</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="3">
        <widget class="QPushButton" name="pushButton_CreateFQ">
         <property name="text">
          <string>生成封签</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QLineEdit" name="lineEdit_FQInfo"/>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="lineEdit_QRcode"/>
       </item>
       <item row="3" column="2">
        <widget class="QLineEdit" name="lineEdit_FHInfo"/>
       </item>
       <item row="2" column="2">
        <widget class="QDateEdit" name="dateEdit_date"/>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="lineEdit_QBInfo"/>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>日期：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>信息：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>券别：</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>复核：</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>复核2：</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLineEdit" name="lineEdit_FHInfo2"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QKXSlipTest.qrc"/>
 </resources>
 <connections/>
</ui>
